packages/manager/apps/identity-access-management/src/pages/manageTokens/components/ManageTokenBreadcrumb.component.tsx

```diff
@@ -0,0 +1,42 @@
+import { useTranslation } from 'react-i18next';
+import {
+  OdsBreadcrumb,
+  OdsBreadcrumbItem,
+} from '@ovhcloud/ods-components/react';
+import { useNavigationGetUrl } from '@ovh-ux/manager-react-shell-client';
+
+export function ManageTokensBreadcrumb() {
+  const { t } = useTranslation('manage-tokens');
+  const { data: iamHref } = useNavigationGetUrl(['', '/iam', {}]);
+  const { data: identitiesHref } = useNavigationGetUrl([
+    '',
+    '/iam/identities',
+    {},
+  ]);
+  const { data: localUserHref } = useNavigationGetUrl([
+    '',
+    '/iam/identities/users',
+    {},
+  ]);
```

Comment:
You could use derivate state
```suggestion
  const { data: iamHref } = useNavigationGetUrl(['', '/iam', {}]);
  const identitiesHref  = `${iamHref}/identities`;
  const localUserHref = `${identitiesHref}/users`
 
```